### PR TITLE
JBPM-6243 - Include support for running the workbench using a standal…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…one controller

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/kiegroup/droolsjbpm-integration/pull/1359

**Comment by cristianonicolai** [comment]
fyi @nmirasch @bsremac @sutaakar @jhrcek

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/utils/MVELEvaluatorProducer.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+package org.kie.workbench.common.screens.server.management.backend.utils;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Produces;
+
+import org.kie.soup.project.datamodel.commons.util.MVELEvaluator;
+import org.kie.soup.project.datamodel.commons.util.RawMVELEvaluator;
+
+@Dependent
+public class MVELEvaluatorProducer {
```
Comment (mswiderski) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/utils/MVELEvaluatorProducer.java]: what is this class used for?

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/main/java/org/kie/workbench/common/screens/server/management/backend/utils/ControllerUtils.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    public static String getControllerURL(){
+        return System.getProperty(KIE_SERVER_CONTROLLER);
+    }
+
+    public static String getControllerUser(){
+        return System.getProperty(CFG_KIE_CONTROLLER_USER, "kieserver");
+    }
+
+    public static String getControllerPassword(){
+        return System.getProperty(CFG_KIE_CONTROLLER_PASSWORD, "kieserver1!");
```
Comment (sutaakar) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/main/java/org/kie/workbench/common/screens/server/management/backend/utils/ControllerUtils.java]: Just an improvement idea - would it be possible to allow loading controller password using Keystore? Something like what is already done in https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-common/src/main/java/org/kie/server/common/KeyStoreHelperUtil.java#L39

Comment (cristianonicolai) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/main/java/org/kie/workbench/common/screens/server/management/backend/utils/ControllerUtils.java]: @sutaakar sure, that should be possible.

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/utils/MVELEvaluatorProducer.java (hunk 1 above)
Comment (cristianonicolai) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/utils/MVELEvaluatorProducer.java]: @mswiderski this is only so it satisfies the CDI dependency in the war that is created for the Arquillian tests. The actual CDI producer for this class is only available in drools-wb, which would not be possible to add as a dependency in here, that's why is only a mock to say.

**Pull request merged**